README.md

```diff
@@ -96,61 +91,85 @@ our [official documentation](https://docs.conjur.org/Latest/en/Content/Developer
+  or `SslVerificationMode.CA_BUNDLE` mode
+
+#### Step 3. Storing credentials
+
+The client uses a credentials provider called `CredentialStores` which inherit from `CredentialsStoreInterface`. This
+approach enables storing the credentials in a safe location, and providing the credentials to the client on demand.
 
-A configuration file called `.conjurrc` is used to hold details required to communicate to the
-Conjur server. You can provide these details needed to open a connection to the Conjur endpoint in
-this file instead of passing them in (`url`, `account`, and `ca_bundle`)  
-during initialization of the Client.
+We provide the user with `CredentialStoreFactory` which create such Credential stores.
 
-The `.conjurrc` file should be saved to your home directory and should contain `conjur_url`,
-`conjur_account`, and`cert_file`.
+By default, the `CredentialStoreFactory` favors saving credentials (login ID and password) to the system's
+credential store. If we do not support the detected credential store, or the credential stoe is not accessible, the credentials are
+written to a configuration file, `.netrc`, in plaintext.
+
+Example of usage:
+
+#####First connection to Conjur:
 
 ```
-# .conjurrc
----
-cert_file: /Users/someuser/conjur-server.pem
-conjur_account: someaccount
-conjur_url: https://conjur-server
+credentials = CredentialsData(login=username, password=api_key, machine=conjur_url)
+
+credentials_provider = CredentialStoreFactory.create_credential_store()
+
+credentials_provider.save(credentials)
+
+del credentials
 ```
 
-#### Storing credentials
+Note: The password should be in the form of the api_key.
 
-When using the SDK, you can keep credentials in the system's native credential store instead of
-passing them in during initialization of the Client. By default, the Client will favor saving
-credentials (login ID and password) to the system's credential store. If the detected credential
-store is not one we support or is not accessible, the credentials will be written to a configuration
-file, `.netrc`, in plaintext.
+##### Already connected:
 
-If written to the `.netrc`, it is strongly recommended that you log out when not using the Conjur
-CLI. This removes the credentials from the `.netrc` file.
+If a prior connection has been made by the SDK or the CLI with your username and account, then the credentials already
+stored in the credentials store. In that case, we only need to get the credentials store using `CredentialStoreFactory`
 
-The `.netrc` file or (`_netrc` for Windows environments) contains credentials needed to log in to
-the Conjur endpoint and should consist of 'machine', 'login', and 'password'.
+```
+credentials_provider = CredentialStoreFactory.create_credential_store()
+```
+
+The `.netrc` file or (`_netrc` for Windows environments) contains credentials needed to log in to the Conjur endpoint
```

Comment:
we use Conjur Server and Conjue endpoint interchangeably in this doc. If they are the same thing, we should make them the same.
What has come up with the Cloud docs is that Conjur is an endpoint, so if we can already align to "Conjur endpoint" or just "Conjur", that would be best
ie "... to log in to Conjur..."